main.py

```diff
@@ -10,36 +10,30 @@
 port = '8989' #YOUR SONARR PORT
 #END
 
-ignore_default_episdode_name = True # Ignore files where the Episode uses default episode naming?
-scrub__and__strings = True # scrub 'and' from file name strings. Fixes & vs 'and' mismatches
+IGNORE_DEFAULT_EPISODE_NAME = True # Ignore files where the Episode uses default episode naming?
```

Comment:
All caps differentiates between static globals and normal variables.

Reply:
I don't know how standard this is, I've seen it a few places though. My editor (vscode) highlights them differently which makes them easy to differentiate.

Reply:
Nice, I'll give this approach a go